packages/config/config/devices/0x027a/zen27_2.8.json

```diff
@@ -440,11 +439,11 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Report each brightness level to hub when physical / Z-Wave control is disabled for physical dimming (only report final level if physical / Z-Wave control is enabled)",
+					"label": "Send each brightness level (Z-Wave disabled); Final level (Z-Wave enabled)",
 					"value": 0
 				},
 				{
-					"label": "Report final brightness level only for physical dimming, regardless of the physical / Z-Wave control mode",
+					"label": "Send final brightness level regardless of physical/Z-Wave mode",
```

Comment:
Should be worded similar to my previous suggestions, but I don't understand what this option does.

Reply:
Clarification from the Zooz docs.  Essentially, it allows for the switch to report each dimming level from the switch when holding down the paddle when local control is disabled. Allows for soft dimming of smart bulbs.